Source/Applications/Wave Demo Apps/UpdateWAVMetaData/Program.cs

```diff
@@ -109,12 +94,12 @@ static int Main(string[] args)
                     if (Convert.ToInt32(connection.ExecuteScalar(database.ParameterizedQueryString("SELECT COUNT(*) FROM Device WHERE Acronym = {0}", "acronym"), acronym)) == 0)
                     {
                         parameterizedQuery = database.ParameterizedQueryString("INSERT INTO Device(NodeID, Acronym, Name, ProtocolID, FramesPerSecond, " +
-                            "MeasurementReportingInterval, ConnectionString, Enabled) VALUES(" + nodeIDQueryString + ", {0}, {1}, {2}, {3}, {4}, {5}, {6})",
+                            "MeasurementReportingInterval, ConnectionString, Enabled) VALUES({7}, {0}, {1}, {2}, {3}, {4}, {5}, {6})",
                             "acronym", "name", "protocolID", "framesPerSecond", "measurementReportingInterval",
                             "connectionString", "enabled");
```

Comment:
You need to also include a name for parameter 7. Probably, you should add 7 at the end and insert the name before `acronym` like so...
```suggestion
                            "MeasurementReportingInterval, ConnectionString, Enabled) VALUES({0}, {1}, {2}, {3}, {4}, {5}, {6}, {7})",
                            "nodeID", "acronym", "name", "protocolID", "framesPerSecond", "measurementReportingInterval",
                            "connectionString", "enabled");
```